packages/gatsby-theme-newrelic/src/components/Callout.js

```diff
@@ -43,8 +62,9 @@ const Callout = ({ title, variant, children }) => {
           css={css`
             font-size: 0.75rem !important;
             text-transform: uppercase;
-            color: var(--heading-text-color);
+            /* color: var(--heading-text-color); */
```

Comment:
We can probably delete this line.
```suggestion
```

Reply:
oops. made the change but didn't commit it. thanks!